worker/cli.py

```diff
@@ -82,6 +82,37 @@ def extract_pose_points(pose):
 
     return new_pose
 
+def get_first_frame_pose_prompts(input_file):
+    capture = cv2.VideoCapture(input_file)
+    frame_number = 0 # to note the 1st frame with a person is detected 
+    poses = None 
+    confs = None
+
+    person_class_id = [id for id, name in model.names.items() if name == "person"][0] # to get class id of "person"
+    
+    while True:
+        success, frame = capture.read()
+        if not success:
+            break
+
+        result = model.predict(source=frame, device='cpu', classes=[person_class_id], verbose=False)[0] # 1st frame
```

Comment:
I don't think the specifying the person class is necessary here as this is the yolo pose model which AFAIK only looks for persons anyway. So for improved simplicity, I would recommend not specifying the person class explicitly